core/consensus/slots/Cargo.toml

```diff
@@ -6,7 +6,7 @@ description = "Generic slots-based utilities for consensus"
 edition = "2018"
 
 [dependencies]
-codec = { package = "parity-codec", version = "3.2" }
+codec = { package = "parity-codec", version = "4.1" }
```

Comment:
Maybe not now but: 
don't we want to put some guidelines on import names? sometimes it can be confusing. I bet you can find the crate `core/sr-primitives` along the code base with names: 
- `primitive`
- `sr-primitives`
- `substrate-primitives`
- ...

Reply:
I like the idea, though it will be hard to apply strict rules everywhere because we have a lot of similar names (for example "primitives").